modules/network/variables.tf

```diff
@@ -37,7 +37,7 @@ variable "autostart" {
 variable "mode" {
   description = "The network mode"
   type        = string
-  default     = "none"
+  default     = "nat"
```

Comment:
Do we need NAT on this? If we have the macvtap interfaces in dev then `none` is fine. this doesn't hurt anything, just might be redundant. Perhaps are we removing the macvtap in local dev and just doing nat for simplification?

Reply:
I had nat for a case where I did not have macvtap enabled and it sort of lingered. It's still dev.